src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -1058,11 +1192,11 @@ def forward(
 
         # normalized features
         image_embeds = image_embeds / torch.linalg.norm(image_embeds, ord=2, dim=-1, keepdim=True)
-        text_embeds = text_embeds / torch.linalg.norm(text_embeds, ord=2, dim=-1, keepdim=True)
+        text_embeds_norm = text_embeds / torch.linalg.norm(text_embeds, ord=2, dim=-1, keepdim=True)
```

Comment:
Also no need for `text_embeds_norm`, can be just `text_embeds`

Reply:
We need to return unnormalized text_embeds if `return_base_embeds=True` though